### PR TITLE
ajuste-style

### PR DESCRIPTION
**Pull request by edwinkay**
solo se hizo cambios en css

### CODE REVIEW

File: src/components/my-images/Photos.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import foto1 from '../../image/foto1.jpeg'
+import foto2 from '../../image/foto2.jpeg'
+import foto3 from '../../image/foto3.jpeg'
+import './Photos.scss'
+
+function Photos() {
```
Comment (chato1337) [inline src/components/my-images/Photos.jsx:7]: use arrow functions for create new compoents

File: src/components/perfil.jsx [hunk 2]
```diff
@@ -45,37 +45,37 @@ function Perfil() {
                     <p className='name-perfil acerca'>Soy una persona apasionada por las ventas, tengo 10 años de experiencia conociendo el mundo del comercio. Me gusta el liderazgo y llevar a mi equipo a lograr los objetivos que nos pongan en el camino. Lo que mas me gusta es viajar y compartir tiempo con mi familia y amigos.</p>
                 </div>
 
-                <div className='contenedor-base'>
-                    <Things />  
-                </div>
-                
-
```
Comment (chato1337) [inline src/components/perfil.jsx:52]: avoid use more than two jump lines

**Review by chato1337 — CHANGES_REQUESTED** [review]

File: src/components/my-images/Photos.jsx (hunk 1 above)
Comment (edwinkay) [reply src/components/my-images/Photos.jsx]: fixed

File: src/components/perfil.jsx (hunk 2 above)
Comment (edwinkay) [reply src/components/perfil.jsx:52]: fixed

**Review by chato1337 — APPROVED** [review]
LGTM

**Pull request merged**